### ISSUE TITLE
supCon-loss

### ISSUE THREAD

**Issue by topDreamer** [deeplearning-wisc/knn-ood#1]
Good work! Could you please tell me where is the supCon-loss in the code?

**Comment by sunyiyou** [comment]
We use the pretrained model directly in this repo. The training scripts are available here: https://github.com/HobbitLong/SupContrast
